aws_lambda_powertools/logging/types.py

```diff
@@ -0,0 +1,10 @@
+from typing import Any, Dict, Iterable, Union
+
+from typing_extensions import TypedDict
+
+class TypedLog(TypedDict, total=False):
+    level: Union[str, int]
```

Comment:
```suggestion
    level: str
```

we're controlling the logging level so in this case it'll always be string. The `Logger(level=20)` will be transformed into `INFO` as a string later.